ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/v2/components/GuideContentComponent.vue

```diff
@@ -40,12 +40,10 @@ export default {
         result = chapters.filter(({ _id }) => hackstackCourseIds.includes(_id))
       } else if (this.product === 'junior') {
         result = chapters.filter(({ _id }) => juniorCourseIds.includes(_id))
-      } else {
-        if (utils.isCodeCombat) {
-          result = chapters.filter(({ _id }) => !hackstackCourseIds.includes(_id) && !juniorCourseIds.includes(_id) && !ozCourseIds.includes(_id))
-        } else {
-          result = chapters.filter(({ _id }) => !hackstackCourseIds.includes(_id) && !juniorCourseIds.includes(_id))
-        }
+      } else if (this.product === 'ozaria') {
+        result = chapters.filter(({ _id }) => ozCourseIds.includes(_id))
+      } else if (this.product === 'codecombat') {
```

Comment:
probably a good idea to set result to [] initially or use else otherwise next line result.map would fail